### ISSUE TITLE
Incorrect margin adjustments in landscape

### ISSUE THREAD

**Issue by lohenyumnam** [SwiftKickMobile/SwiftMessages#185]
Hello guys, 

I was testing the cool library. and I don't know if its a bug. when using **layout** as **statusLine** with no tab bar controller, in landscape mode the message is under the home bar of iPhone X  as shown in the image below

![simulator screen shot - iphone x - 2018-05-15 at 22 01 54](https://user-images.githubusercontent.com/17508663/40070848-dd64af30-588c-11e8-9142-39660c07a59f.png)

Here is the code I used 

``` Swift
        let view = MessageView.viewFromNib(layout: .statusLine)
        view.configureTheme(backgroundColor: UIColor.blue, foregroundColor: .white)

        view.configureContent(body: "Hello Lohen Yumnam")
        
        
        var config = SwiftMessages.Config()
        config.presentationStyle = .bottom // Slide up from the bottom.
        config.shouldAutorotate = true
        config.interactiveHide = true
        config.duration = .forever
        config.presentationContext = .window(windowLevel: UIWindowLevelStatusBar)
        
        // Show the message.
        SwiftMessages.show(config: config, view: view)

```

**Comment by wtmoose** [comment]
Thanks for bringing this to my attention. It definitely shouldn't do that. I'll take a look.

**Comment by wtmoose** [comment]
This fix is available in 4.1.3
